docs/kuberneter.md

```diff
@@ -0,0 +1,302 @@
+| networking.k8s.io/v1               | IngressClass <br/>Ingress <br/>NetworkPolicy                                                                                      |
+| policy/v1                          | PodDisruptionBudget                                                                                                     |
+| rbac.authorization.k8s.io/v1       | ClusterRoleBinding <br/>ClusterRole <br/>RoleBinding <br/>Role                                                                         |
+| apps/v1beta1                       | Deployment <br/>StatefulSet                                                                                                  |
+| apps/v1beta2                       | DaemonSet <br/>Deployment <br/>ReplicaSet S<br/>tatefulSet                                                                             |
+| batch/v1beta1                      | CronJob                                                                                                                 |
+| networking.k8s.io/v1beta1          | IngressClass <br/>Ingress                                                                                                    |
+| policy/v1beta1                     | PodDisruptionBudget <br/>PodSecurityPolicy                                                                                   |
+| rbac.authorization.k8s.io/v1alpha1 | ClusterRoleBinding <br/>ClusterRole <br/>RoleBinding <br/>Role                                                                         |
+| rbac.authorization.k8s.io/v1beta1  | ClusterRoleBinding <br/>ClusterRole <br/>RoleBinding <br/>Role                                                                         |
+
+
+## Running KICS to scan runtime K8s cluster
+
+When running KICS using a kuberneter path, the resources are imported using the credentials set as environment variables in Kubernetes format to the current working directory in a new folder named `kics-extract-kuberneter` following the above-described structure.
+KICS will then run a scan on these local files.
+
+If the flag `-o, --output-path` is passed the folder `kics-extract-kuberneter` will be generated in the reports directory instead.
+
+### Imported Resources tree structure:
+
+```
+ ▾ kics-extract-kuberneter/
+    ▾ {apiGroup}/
+        ▾ {apiVersion}/
+            ▾ {kind}.yaml
+```
+
+### Run KICS to scan runtime K8s cluster with Docker
+
+To run KICS Kuberneter with Docker, you can simply pass the K8s Credentials that were set as environment variables to the docker run command and use the kuberneter path syntax
+
+Examples:
+
+
+#### Using Config File
+```sh
+docker run -v <credentials_path>:/credentials -v ${PWD}:/path/ -e K8S_CONFIG_FILE=/credentials/<config-file-name> checkmarx/kics:latest scan -p "kuberneter::*:*:*" -v --no-progress -o /path/results
+```
+
+#### Using Service Account Token
+```sh
```

Comment:
Something looks off and repeated from this point to the end...

Reply:
My bad. It was not supposed

Reply:
My bad. It was not supposed. Sorry.